lib/travis/api/v3/queries/remote_log.rb

```diff
@@ -1,32 +1,25 @@
 module Travis::API::V3
-  class Queries::Log < RemoteQuery
+  class Queries::RemoteLog < RemoteQuery
+    def main_type
+      'log'
+    end
 
     def find(job)
       @job = job
-      #check for the log in the Logs DB
-      log = logs_model.find_by_job_id(@job.id)
+      log = Travis::RemoteLog.find_by_job_id(@job.id)
       raise EntityMissing, 'log not found'.freeze if log.nil?
-      #if the log has been archived, go to s3
+      # if the log has been archived, go to s3
       if log.archived?
         content = fetch.first
         raise EntityMissing, 'could not retrieve log'.freeze if content.nil?
         body = content.body.force_encoding('UTF-8') unless content.body.nil?
```

Comment:
Should we build a RemoteLog part for this info, or assign the content to the RemoteLog as you suggest and it will have no RemoteLogParts? I'm not sure which is more consistent from a user's perspective about the return types (like should a log that is returned always have log parts? I think that is the current assumption).